cmd/nomos/migrate/migrate.go

```diff
@@ -45,23 +45,28 @@ import (
 )
 
 const (
-	migrateDir       = "nomos-migrate"
-	rootSyncYamlFile = "root-sync.yaml"
-	cmOrigYAMLFile   = "cm-original.yaml"
-	cmMultiYAMLFile  = "cm-multi.yaml"
+	monorepoMigrateDir         = "nomos-migrate-monorepo"
+	configManagementMigrateDir = "nomos-migrate-configmanagement"
+	rootSyncYamlFile           = "root-sync.yaml"
+	cmOrigYAMLFile             = "cm-original.yaml"
+	cmOperatorYAMLFile         = "config-management-operator.yaml"
+	cmMultiYAMLFile            = "cm-multi.yaml"
 
 	updatingConfigManagement    = "Updating the ConfigManagement object ..."
-	waitingForRootSyncCRD       = "Waiting for the RootSync CRD to be established ..."
+	waitingForRootSyncCRD       = "Waiting for ConfigSync CRDs to be established ..."
```

Comment:
Suggest renaming the variable to match the content.

Reply:
done